docs/src/conservation.md

```diff
@@ -0,0 +1,16 @@
+# Conservation Checks
+
+If the model is a physically closed system (e.g., in the `slabplanet` configuration with free slip conditions), it should conserve mass (including water), energy and momentum. The conservation checker logs global conservation.
+
+Only energy and water are currently implemented. 
+
+Note that kinetic energy is not included in the calculation of the global energy, reflecting the formulation on `ClimaAtmos`, which assumes that kinetic energy is negligible in comparison with the moist static energy components. 
```

Comment:
I think this is a bit confusing because e.g. when you calculate the total energy of atmosphere using `e_tot`, it includes kinetic energy. Maybe rephrase it as something like `Kinetic energy transfer between atmosphere, land, and ocean is not included`?

Reply:
That was rather misleading, thanks for pointing it out! It was meant to refer to the KE contribution being neglected in all diffusive fluxes in the atmos interior, so this is consistent with these BCs. In any case, since we won't be needing this function (except perhaps for diagnostics?) in the foreseeable, I've decided to remove it for now. :P